src/jdk.jdwp.agent/unix/native/libjdwp/linker_md.c

```diff
@@ -89,14 +98,12 @@ dbgsysBuildLibName(char *holder, int holderlen, const char *pname,
 {
     const int pnamelen = pname ? strlen(pname) : 0;
 
-    *holder = '\0';
-    // Quietly truncate on buffer overflow.  Should be an error.
-    if (pnamelen + (int)strlen(fname) + 10 > holderlen) {
-        return;
-    }
-
     if (pnamelen == 0) {
-        (void)snprintf(holder, holderlen, "lib%s." LIB_SUFFIX, fname);
+        size_t result_len = (size_t)snprintf(holder, holderlen, "lib%s." LIB_SUFFIX, fname);
+        if (result_len >= holderlen) {
```

Comment:
Please change the type of `result_len` to `int` to match `holderlen` (and remove the cast).
```
src/jdk.jdwp.agent/unix/native/libjdwp/linker_md.c: In function ‘dbgsysBuildLibName’:
src/jdk.jdwp.agent/unix/native/libjdwp/linker_md.c:103:24: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
         if (result_len >= holderlen) {
                        ^~
```
I expect a similar error on Windows.

Reply:
Odd. This code compiles fine for me on linux, and the windows code in the link below was introduced months ago. Will make the change anyway.

https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/107

Reply:
Done. Running checks.